VAMobile/src/components/Templates/HeaderBanner.tsx

```diff
@@ -161,6 +162,18 @@ const HeaderBanner: FC<HeaderBannerProps> = ({
     borderBottomWidth: bannerDivider ? theme.dimensions.borderWidth : 0,
     borderBottomColor: 'menuDivider',
     mb: !dividerMarginBypass && bannerDivider ? theme.dimensions.standardMarginBetween : 0,
+    style: {
+      shadowColor: colors.black,
+      ...Platform.select({
+        ios: {
+          shadowOffset: { width: 0, height: 6 },
+          shadowOpacity: 0.4,
+        },
+        android: {
+          elevation: 8,
+        },
+      }),
+    },
```

Comment:
A few things I noticed:
1. This will impact all templates, which will affect nearly all screens in the app. This should only be impacting the `CategoryLanding` screens. One option is to add a prop to the `HeaderBanner` that conditionally adds this shadow style, and use that in the `CategoryLanding` template
2. The shadow looks darker than what's in the Figma file

Reply:
@theodur for #2 , I did notice that as well but just entered as noted in the figma. But for matching sake I'll change opacity to 0.1

Reply:
fixed!